src/mame/misc/marywu.cpp

```diff
@@ -21,7 +21,7 @@
 #include "machine/i8279.h"
 #include "sound/ay8910.h"
 #include "speaker.h"
-
+#include "machine/nvram.h"
 #include "marywu.lh"
```

Comment:
You should put the header at the proper place (alphabetical order within the headers from src/devices).  Don’t remove blank lines separating sections.